app/components/references/notification-card-list.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+    <div>
+        <div 
+            v-for="notification in notificationList"
+            :key="notification.symbol"
+            class="card" 
+            style="margin-bottom:4px">
+            <div class="card-body" style="padding:12px;font-size:0.9em">
+                <notification-card :notification="notification"/>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+import NotificationCard from '~/components/references/notification-card.vue'
+import Api from '~/components/meetings/api.js';
+import _ from 'lodash';
+import solr from '~/util/solr.js';
+
+export default {
+    name: 'NotificationCardList',
+    components: { NotificationCard },
+    props: {
+        notifications: {
+            type: Array,
+            default: () => []
+        }
+    },
+    data() {
+        return {
+            api: new Api(),
+            notificationList: [],
+        }
+    },
+    created:refresh,
+    watch: {
+        notifications: refresh
+    },
+    methods: {
+        lookupNotifications
+    }
+}
+
+async function refresh() {
+    const codes = this.notifications.filter(c => !!c);
+
+    const notificationList = await this.lookupNotifications(codes);
+    
+    this.notificationList = codes.map(c => notificationList.find(n => n.symbol === c) || {symbol: c, files: []})
+}
+
+async function lookupNotifications(codes) {
+    if(!codes || codes.length === 0) return [];
+
+    const q = `symbol_s (${codes.map(solr.escape).join(' or ')})`
```

Comment:
missing `:` in query
```suggestion
    const q = `symbol_s : (${codes.map(solr.escape).join(' ')})`
```

Reply:
Issue fixed.